Error during installation

Hi all,
I’m unsure this tool can still work with latest WriteFreely versions, but the doc mentions obsolete installation commands, as the `go` cli asks to use `go install` instead of `go get` : 
```
go: go.mod file not found in current directory or any parent directory.
        'go get' is no longer supported outside a module.
        To build and install a command, use 'go install' with a version,
        like 'go install example.com/cmd@latest'
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
```
plus it requires me to specify « a version when current directory is not in a module » :
`go install github.com/writeas/wf-migrate/cmd/wfimport@latest`


But above all, the installation script mentions it can’t find some of the structures and method/functions mentioned in the `import.go` file, and I think the installation fail there. (I don’t find the `wfmigrate` command anywhere after that) :
```
➜  ~ go install github.com/writeas/wf-migrate/cmd/wfimport@latest                                                                          
go: finding module for package github.com/writeas/go-writeas
go: finding module for package github.com/howeyc/gopass
go: found github.com/howeyc/gopass in github.com/howeyc/gopass v0.0.0-20210920133722-c8aef6fb66ef
go: found github.com/writeas/go-writeas in github.com/writeas/go-writeas v1.1.0
go: finding module for package github.com/writeas/impart
go: finding module for package golang.org/x/crypto/ssh/terminal
go: finding module for package code.as/core/socks
go: found golang.org/x/crypto/ssh/terminal in golang.org/x/crypto v0.28.0
go: found code.as/core/socks in code.as/core/socks v1.0.0
go: found github.com/writeas/impart in github.com/writeas/impart v1.1.1
# github.com/writeas/wf-migrate
go/pkg/mod/github.com/writeas/wf-migrate@v0.0.0-20190609185100-876a57e8eedd/import.go:14:3: unknown field Slug in struct literal of type writeas.PostParams
go/pkg/mod/github.com/writeas/wf-migrate@v0.0.0-20190609185100-876a57e8eedd/import.go:20:3: unknown field Created in struct literal of type writeas.PostParams
go/pkg/mod/github.com/writeas/wf-migrate@v0.0.0-20190609185100-876a57e8eedd/import.go:21:3: unknown field Updated in struct literal of type writeas.PostParams
go/pkg/mod/github.com/writeas/wf-migrate@v0.0.0-20190609185100-876a57e8eedd/import.go:21:18: p.Updated undefined (type writeas.Post has no field or method Updated)
```

But I’m not familiar at all with the go environment, so maybe I’m missing something.



(Big up for Writeyfreely development, it’s a great piece of software !)

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # wf-migrate
 
-[![GoDoc](https://godoc.org/github.com/writeas/wf-migrate?status.svg)](https://godoc.org/github.com/writeas/wf-migrate)
+[![GoDoc](https://pkg.go.dev/badge/github.com/writefreely/wf-migrate?status.svg)](https://pkg.go.dev/github.com/writefreely/wf-migrate)
 
 wf-migrate provides helper functions and a command-line utility for migrating posts between [WriteFreely](https://writefreely.org) instances.
 
@@ -9,7 +9,7 @@ wf-migrate provides helper functions and a command-line utility for migrating po
 Install the command-line utility with:
 
 ```
-go get github.com/writeas/wf-migrate/cmd/wfimport
+go install github.com/writefreely/wf-migrate/cmd/wfimport
 ```
 
 `wfimport` takes a username `-u`, optional WriteFreely instance hostname `-h`, and the filename of the JSON data you want to import.

--- a/cmd/wfimport/main.go
+++ b/cmd/wfimport/main.go
@@ -5,8 +5,8 @@ import (
 	"flag"
 	"fmt"
 	"github.com/howeyc/gopass"
-	"github.com/writeas/go-writeas"
-	"github.com/writeas/wf-migrate"
+	"github.com/writeas/go-writeas/v2"
+	"github.com/writefreely/wf-migrate"
 	"io/ioutil"
 	"os"
 )

--- a/go.mod
+++ b/go.mod
@@ -1,0 +1,16 @@
+module github.com/writefreely/wf-migrate
+
+go 1.23.0
+
+require (
+	github.com/howeyc/gopass v0.0.0-20210920133722-c8aef6fb66ef
+	github.com/writeas/go-writeas/v2 v2.1.0
+)
+
+require (
+	code.as/core/socks v1.0.0 // indirect
+	github.com/writeas/impart v1.1.1 // indirect
+	golang.org/x/crypto v0.28.0 // indirect
+	golang.org/x/sys v0.26.0 // indirect
+	golang.org/x/term v0.25.0 // indirect
+)

--- a/import.go
+++ b/import.go
@@ -1,6 +1,6 @@
 package wfmigrate
 
-import "github.com/writeas/go-writeas"
+import "github.com/writeas/go-writeas/v2"
 
 type Import struct {
 	writeas.User

